src/components/Navbar/NavbarDesktop.tsx

```diff
@@ -23,13 +21,13 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
-export const NavbarDesktop = ({ className, withoutBlogPage, ...props }: NavbarDesktopProps) => {
+export const NavbarDesktop = ({ className, ...props }: NavbarDesktopProps) => {
   const classes = useStyles();
 
   return (
     <Box {...props} className={clsx(className, classes.root)}>
       <MenuList variant="menu" className={classes.menu}>
-        {renderNavItems(undefined, withoutBlogPage)}
+        {renderNavItems(undefined)}
```

Comment:
It's not necessary for this PR, but we should fix the logic of this function so we wouldn't pass `undefined` here.

Reply:
I did make the parameter optional so we no longer have to pass undefined 👍